plugin.json

```diff
@@ -1,19 +1,29 @@
 {
-    "name": "<TODO: your plugin name>",
-    "url": "<TODO: Your Plugin URL here>",
-    "description": "<TODO: Your Plugin description here>",
-    "main": "index.js",
+    "name": "Calendly Event Enricher",
+    "url": "https://github.com/PostHog/calendly-app",
+    "description": "Enriches Calendly events with additional event, owner and attendee information",
```

Comment:
Can we make this a bit more descriptive? I had difficulty understanding what the plugin does without going through the code.

e.g. `Syncs React-Calendly event into posthog, including event, owner and attendee information`

For the rest, all good!